packages/manager/modules/pci/src/projects/project/storages/databases/translations/Messages_fr_FR.json

```diff
@@ -29,5 +29,7 @@
   "pci_database_node_type_memory": "{{memorySize}} RAM",
   "pci_database_node_type_storage": "{{storageSize}} de stockage SSD",
   "pci_database_node_type_storage_range": "de {{storageMinSize}} à {{storageMaxSize}} de stockage SSD",
-  "pci_database_node_type_label": "Modèle de nœud"
+  "pci_database_node_type_label": "Modèle de nœud",
+  "pci_database_payment_warning_title": "Bases de données Public Cloud - fin de l'essai gratuit",
+  "pci_database_payment_warning_description": "Tout service actif créé en tant que Beta testeur deviendra payant à partir du 01/02/2022. Si vous ne souhaitez pas être facturé, merci de bien vouloir supprimer vos services d'ici le 31/01/2022."
```

Comment:
can you reuse your warning date constant here as a translation variable ? would make future updates easier if the date needs to be updated